server/build.gradle

```diff
@@ -29,6 +29,7 @@ base {
 dependencies {
 
   api project(':libs:elasticsearch-core')
+  api project(":libs:elasticsearch-ssl-config")
```

Comment:
I think this should only be `implementation`. see my comment here: https://github.com/elastic/elasticsearch/pull/113803#issuecomment-2674275093